storybook/preview.tsx

```diff
@@ -31,15 +31,25 @@ const decorators = [
       <Story />
     </KaizenProvider>
   ),
-  (Story, props): JSX.Element => {
+  (Story, context): JSX.Element => {
     useEffect(() => {
-      const dir = props.parameters.textDirection ?? props.globals.textDirection
+      const dir =
+        context.parameters.textDirection ?? context.globals.textDirection
       if (document.body.getAttribute("dir") !== dir)
         document.body.setAttribute("dir", dir)
-    }, [props])
+    }, [context])
 
     return <Story />
   },
+  (Story, context) =>
+    context.args.isReversed &&
+    context.parameters.backgrounds.default !== "Purple 700" ? (
```

Comment:
Seems a little strange to look out for specifically "Purple 700", what about an alias string? Otherwise it seems like you could do any colour.
```
parameters: { background: "reversed" }
```

Reply:
Yeah... It's what's been set up tbh. We have a `backgrounds` array (see `storybook/backgrounds.ts`) and that's the key/name they've given it.
I also question whether anyone ever even uses any of the background colours offered (cos I don't). Didn't want to blow up the PR with a config refactor though, but happy to revisit later.

Reply:
Oh it's existing, meh then lol